packages/flutter/test/widgets/scrollable_selection_test.dart

```diff
@@ -493,6 +494,113 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('select to scroll by dragging start selection handle stop scroll when release', (WidgetTester tester) async {
```

Comment:
"stop" => "stops", "release" => "released"

And same for the next test's title as well.